src/guide/views/button/ButtonExample.jsx

```diff
@@ -79,6 +80,17 @@ export default class ButtonExample extends Component {
           </PrimaryButton>
         </Example>
 
+        <Example title="AlertButton">
+          <Text>
+            This button lets the user perform a special action, like
+            &quot;Cancel&quot;. It&apos;s higher in the UI hierarchy than the
+            BasicButton.
+          </Text>
+          <AlertButton>
+            Cancel
```

Comment:
Perhaps `Delete` makes more of an example for this button. And not sure of those descriptions are really needed but ok, just a suggestion.